users/mnil/mnil.h

```diff
@@ -21,39 +21,41 @@
 
 // Layers
 enum layers {
-    _COLEMAK,
+    _COLEMAK = 0,
```

Comment:
This doesn't need to be set.  By default, enums start at 0, and increment up from there. 
```suggestion
    _COLEMAK,
```